Sources/SwiftParser/CharacterInfo.swift

```diff
@@ -70,35 +70,30 @@ extension Unicode.Scalar {
   ) -> Bool {
     let info: Character.Info
     switch self.value {
-    case
-      // '0'-'9'
-      48, 49, 50, 51, 52, 53, 54, 55, 56, 57:
+    case  // '0'-'9'
+    48, 49, 50, 51, 52, 53, 54, 55, 56, 57:
```

Comment:
@allevato Do you know if this is a bug in swift-format or whether this formatting is intentional? 

We can always turn off formatting for some of these lines with `// swift-format-ignore` as well.

Reply:
This looks like a bug; we have some logic that tries to determine whether a comment is an end-of-line comment or some other inline comment so that we can do something that looks reasonable with it, and it looks like it's falling over in this case. At the _very least_ we shouldn't be dedenting that next line.